core/trino-parser/src/main/java/io/trino/sql/tree/SetAuthorizationStatement.java

```diff
@@ -28,13 +28,6 @@ public abstract class SetAuthorizationStatement
     private final QualifiedName source;
     private final PrincipalSpecification principal;
 
-    public SetAuthorizationStatement(QualifiedName source, PrincipalSpecification principal)
```

Comment:
I think that in such cases we should have an automated check that verifies whether the method is actually used